actions/platform/service_account/create_service_account.py

```diff
@@ -0,0 +1,113 @@
+from typing import Any, Text, Dict, List
+import re
+
+from rasa_sdk import Action, Tracker, FormValidationAction
+from rasa_sdk.executor import CollectingDispatcher
+from rasa_sdk.events import SlotSet, ActiveLoop
+from rasa_sdk.types import DomainDict
+
+from common import logging
+from common.config import app
+from actions.slot_match import FuzzySlotMatch, FuzzySlotMatchOption, resolve_slot_match
+
+logger = logging.initialize_logging()
+
+NEW_OR_EXISTING = "new_or_existing"
+SERVICE_ACCOUNT_NAME = "service_account_name"
+SERVICE_ACCOUNT_DESCRIPTION = "service_account_description"
+CREATE_ANOTHER_SERVICE_ACCOUNT = "create_another_service_account"
+
+new_or_existing_choice = FuzzySlotMatch(
+    "new_or_existing",
+    [
+        FuzzySlotMatchOption("new"),
```

Comment:
What if they say "one I already have" just to trip us up haha. Mostly kidding, but it doesn't seem robust, but maybe thats all good.

Reply:
Yeah, you can potentially say a lot of things like this under `new`. I am skeptical if we are able to capture all the edge case for this one. We can add more in the QE/Feedback stage if necessary. Not making any changes for this one yet.